lib/ansible/modules/cloud/amazon/iam_user_facts.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+
+# -*- coding: utf-8 -*-
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: iam_user_facts
+short_description: Gather IAM user(s) facts in AWS
+description:
+  - Gather IAM user(s) facts in AWS
+version_added: "2.4"
+author: "Constantin Bugneac, (@Constantin07)"
+options:
+  name:
+    description:
+     - The name of the IAM user to look for.
+    required: false
+  group:
+    description:
+     - The group name name of the IAM user to look for. Mutually exclusive with C(path).
+    required: false
+  path:
```

Comment:
Can you add an example or explanation of group vs. path, and when each would be used?